release-content/0.14/release-notes/11535_Query_Joins.md

```diff
@@ -1 +1,21 @@
-TODO
+Queries can now be combined, returning the data for entities that are contained in both queries.
+
+```rust
+fn socratic_deduction_the_hard_way(mut all_men: Query<&Man>, mut all_mortals: Query<&Mortal>){
```

Comment:
I'm not sure how I feel about this example. It feels a little weird that your combined query is not a proper subset of the original queries. Like in most cases you're going to use this to narrow your query.

Reply:
Fair! I'll try and find something more representative.